docs/guides/multicollateral-warp-route.mdx

```diff
@@ -0,0 +1,178 @@
+# Multicollateral Warp Route Rebalancing
+
+## Overview
+
+A **Multicollateral Warp Route** is a way to bridge multiple existing tokens across different chains while maintaining liquidity. Instead of introducing a new synthetic asset for every transfer, it allows chains to leverage existing token liquidity in a structured manner.
+
+This is useful when expanding to new chains, ensuring users have access to assets that already exist in the ecosystem. The diagram below shows a setup where canonical USDC exists on Base and Arbitrum, with a synthetic warp route created on a new chain.
+
+```mermaid
+flowchart
+    subgraph base
+    BU((User))
+    BWR[USDC Collateral Warp Route]
+    end
+
+    subgraph arbitrum
+    AU((User))
+    AWR[USDC Collateral Warp Route]
+    end
+
+    subgraph newchain
+    NWR[USDC Synthetic Warp Route]
+    Recipient((Recipient))
+    end
+
+    BU -- "transferRemote(newchain,<br>recipient, amount)" --> BWR
+    BWR -. "{recipient, amount}" .-> NWR
+    AU -- "transferRemote(newchain,<br>recipient, amount)" --> AWR
+    AWR -. "{recipient, amount}" .-> NWR
+    NWR -. "2 \* amount" .-> Recipient
+```
+
+### Collateral Imbalances
+
+If more funds flow in one direction, one of the chains in the route can run out of collateral. This _imbalanced flow_ prevents withdrawals until the collateral is covered.
+
+```mermaid
+flowchart
+	subgraph base
+		BWR[USDC Collateral Warp Route]
+	end
+
+	subgraph arbitrum
+		AWR[USDC Collateral Warp Route]
+	end
+
+	subgraph newchain
+		NWR[USDC Synthetic Warp Route]
+		NU((Recipient))
+	end
+
+	NU -- "transferRemote(arbitrum,<br> recipient, 2 * amount)" --> NWR
+	NWR -. "{recipient, 2 * amount}" .-> AWR
+```
+
+To fix this and mantain a good user experience we must perform _collateral rebalancing_, collateral must be moved between chains to restore balance. An aligned actor may assue a traditional Liquidity Provider (LP) role and provide collateral to address these imbalances using their own inventory.
```

Comment:
```suggestion
To fix this and mantain a good user experience we must perform _collateral rebalancing_, collateral must be moved between chains to restore balance. An aligned actor may assume a traditional Liquidity Provider (LP) role and provide collateral to address these imbalances using their own inventory.
```
do we have spellcheck running in CI?

Reply:
"collateral must be moved between chains to restore balance"
this also feels a little imprecise
collateral does not have to be moved "between chains" necessarily, it could come from some local inventory

Reply:
no spellcheck atm, but will do. ty for callout! addressed these both